wpeverest/user-registration
# Feature/profile picture upload field

### All Submissions:






### Changes proposed in this Pull Request:
This Pull request extends the feature of the new profile picture upload field introduced in advanced fields addon by enabling the user to change profile picture from edit profile section and tends to exclude the fields like user password, email confirmation, etc from the profile details section of my account page.

### How to test the changes in this Pull Request:

1. After a user is registered from the form containing a profile picture upload field, navigate to the edit profile section of my account page.
2. The fields like user password, confirm password and confirm email are not displayed in the edit profile section. 
3. There will be the profile picture with which the user is registered and below it, there will be options for removing and either uploading the picture from the computer or capturing from the webcam.
4. Once either of the options is selected, a popup with cropping container is displayed and after cropping is done and on saving the changes will change the profile picture.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Profile picture upload field in advanced fields addon.
